keras/backend/theano_backend.py

```diff
@@ -5,6 +5,11 @@
 from theano.tensor.nnet import conv3d2d
 from theano.printing import Print
 try:
+    import theano.sparse as T_sp
+    HAS_SPARSE = True
```

Comment:
`T_sp` should be renamed to something readable (like `th_sparse_module`). No need for an additional global variable, just set `th_sparse_module` to None if not available.